frontend/views/containers/chatroom/chat-mentions/chat-mentions-utils.js

```diff
@@ -5,12 +5,19 @@ export type DomObject = {
   children?: Array<DomObject>
 }
 
-export function htmlStringToDomObjectTree (htmlString: string): Array<DomObject> {
+export function htmlStringToDomObjectTree (htmlString: any): Array<DomObject> {
```

Comment:
Why was this updated to `any`? What besides `string` can it be? If it can be one of two types, then maybe it should be changed to a union of two types?

Reply:
@taoeffect 
I wanted to fix (or avoid) this Flow error.

![image](https://github.com/okTurtles/group-income/assets/17641213/6ec5c86f-a95e-4ffa-8fbd-81c557abbf18)

Which actually doesn't make sense as `replaceAll` is [a valid method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) of the string type.

Reply:
@SebinSong if flow is being dumb then you can add a comment above the broken line like this:

```js
// $FlowFixMe[prop-missing]
```

It's better to get the type information in the parameters right and add a comment like this than use the wrong type information because it could mislead other developers looking at this code into thinking they can pass anything to the function.

Reply:
@SebinSong 
> Which actually doesn't make sense as replaceAll is [a valid method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) of the string type.
This is true but wasn't the case when this older Flow version (0.154.0) was released